packages/cockpit/src/business/requestBusinessCalendar.js

```diff
@@ -1,8 +1,7 @@
 import fetch from 'isomorphic-fetch'
 
-const requestBusinessCalendar = year =>
-  fetch(`https://pagarme.github.io/business-calendar/data/brazil/${year}.json`)
-    .then(res => res.json())
+const requestBusinessCalendar = year => fetch(`https://pagarme.github.io/business-calendar/data/brazil/${year}.json`)
```

Comment:
será que aqui não fica melhor assim pra evitar da linha com a url ficar muito grande?
```suggestion
const requestBusinessCalendar = year => (
  fetch(`https://pagarme.github.io/business-calendar/data/brazil/${year}.json`)
    .then(res => res.json())
)
```